bin/MadGraph5_aMCatNLO/cards/production/13TeV/wellnu012j_5f_NLO_FXFX_bias_binned/wellnu012j_5f_NLO_FXFX_bias_binned_madspin_card.dat

```diff
@@ -0,0 +1,21 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+set ms_dir ./madspingrid
+
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+set max_running_process 1
+
+set onlyhelicity True
+decay w+ > ta+ vt
```

Comment:
is madspin needed for real (I don't see the intermediate w in the process card) or to solve a nasty/random error happening otherwise?

Reply:
I'm not actually sure - madspin was also used for a previous inclusive W+Jets aMC@NLO sample so I took this to be consistent with that, I have to say I didn't try without so don't know what happens in that case...

Reply:
Madspin should be used in this case to make sure the proper spinup flag gets written in the LHE file.

(it's needed to ensure the correct polarization for tau decays in pythia 8)